activerecord/lib/active_record/insert_all.rb

```diff
@@ -25,7 +27,7 @@ def keys
       inserts.first.keys.map(&:to_s)
     end
 
-    def updatable_columns
+    def default_updatable_columns
```

Comment:
In the context of this PR, I like the rename of `updatable_columns` to `default_updatable_columns`.